src/exceptions.py

```diff
@@ -18,3 +18,35 @@ def __init__(self, msg: str):
             msg (str): Explanation of the error.
         """
         self.msg = msg
+
+
+class InvalidZoneFileMetadataError(Exception):
```

Comment:
Perhaps it would be more cohesive to define these two exception in bind.py

Reply:
I thought a bit about this and I think that I prefer having them in a separate file like this. There are currently 3 exceptions in the `exceptions.py` file that are all affiliated to `bind.py`. But their code doesn't really add value: it's only an init with a message and some doc/comments. All the interesting information to use them and comprehend the rest of the code is their name.  
In that sense, I'd rather leave them in another file to avoid cluttering `bind.py` which is already starting to get big.

Reply:
No big deal :)